### ISSUE TITLE
Name, phone number, email not displayed on view bookings page

### ISSUE THREAD

**Issue by Richardng12** [SE701ProjectGroup4/git-brunching#236]
**Describe the bug**
When making a booking, and a user goes back to check their bookings, their name, phone number and email are not shown

**To Reproduce**
1. Make a booking for a restaurant, filling out relevant fields
2. Go back to main menu and view your bookings
3. Relevant fields are not shown

**Expected behavior**
All details should be shown on screenshot

**Screenshots**
![image](https://user-images.githubusercontent.com/42757966/79626005-f84fd580-8180-11ea-8598-984cf8c9f02e.png)

**Additional context**

**Comment by ZianYangAng** [comment]
LGMT

**Comment by wryao64** [comment]
We need to change the Reservation table to display the customer's details since the User table is for restaurant owners. Can you please create a new issue for this?

**Comment by wryao64** [comment]
Blocked by #239

**Comment by wryao64** [comment]
Blocked by #242

**Comment by wryao64** [comment]
Fixed by #243
